elftools/dwarf/namelut.py

```diff
@@ -160,9 +160,8 @@ def _get_entries(self):
         offset = 0
         entry_struct = Struct("Dwarf_offset_name_pair",
                 self._structs.Dwarf_offset('die_ofs'),
-                CString('name'))
-        die_ofs_struct = self._structs.Dwarf_offset('die_ofs')
-                
+                If(lambda ctx: ctx['die_ofs'],CString('name')))
```

Comment:
Can you add a comment above `entry_struct =` explaining why this conditional parsing exists and what the expected result is when the offset is 0 or not 0

Reply:
Done.